go/pkg/trainingjober/trainingjober.go

```diff
@@ -0,0 +1,62 @@
+// trainingjober is a package for managing a TrainingJob
```

Comment:
Sorry I don't think `trainingjober` is a suitable name, we need a *Controller* to manager the TrainingJob, so how about rename `TrainingJober` to `TrainingJobController` ?